.github/SUPPORT.md

```diff
@@ -0,0 +1,4 @@
+# Support
+
+GitHub issues are meant for enhancement requests and specific, reproducible bugs, not for general support questions.
```

Comment:
Maybe provide a link to the community support forum on wp.org

Reply:
Done in 58d532e.